test/integration.js

```diff
@@ -1348,6 +1348,11 @@ test('try to revert a deployment and assign the automatic aliases', async t => {
   }
 });
 
+test('try to update now to canary', async t => {
+  const { code } = await execute(['update', '--channel', 'canary', '--yes']);
+  t.is(code, 0);
```

Comment:
Let's add another test to verify that `now --version` includes "canary" in the output.